src/main.tsx

```diff
@@ -20,8 +22,13 @@ AppInfo.initialize()
   .then(() => {
     console.log('AppInfo initialized successfully.');
     initializeDatabase()
-      .then(() => {
+      .then(async () => {
         console.log('Database initialized successfully.');
+        const settings = await NotificationsSettingsService.getSettings();
+        if (settings.enabled) {
+          await notificationScheduler.start();
+        }
```

Comment:
They scheduler should check if the settings are enabled. 


```suggestion
        await notificationScheduler.start();
```